### DISCUSSION TITLE
年間活動内容に応じたトロフィー表示のような機能

### DISCUSSION THREAD

**Discussion by yoshi389111** [increments/qiita-discussions#117] [要望・アイデア・不具合]
先日公開された [Qiitaのいろいろランキング2021](https://qiita.com/Qiita/items/28e1aa0967e607d145fa) を読んでぼんやりおもった、単なる思いつきですが。

主にユーザーのモチベーションアップのために、１年間の活動内容に応じたトロフィーみたいなものを翌年1年間限定でユーザに付与して、ユーザーページで表示可能にする（他人に見せられる）という機能があったらよいのでは、と愚考しました。

ただし、トップ100とかだけだと「自分には無理」と思ってしまいそうなので、もう少し難易度を下げて、例えば「2022年コントリビュート数の上位20%」だとか「2022年執筆記事数の上位10%」とか、ちょっと頑張れば取れそうな感じ、かつもう少し頑張れば一つ上が狙える感じにすると、結果が目に見えて、モチベーションが上がるのではないでしょうか。
（上位20%が難易度低めかどうかはわかりませんが）

特に、現状は人が少なくて、でも増えてほしいような分野のトロフィーがあると、結果的に全体の質もあがるのではないでしょうか。たとえば「編集リクエスト」とか「質問への回答」とかは、上記ランキングを見た感じでは人が少なそう。
（質を上げるためには、「採用された編集リクエスト数」とか「LGTMが付いている回答数」とかの方がよいかもしれませんが）

さらにいえば、こういったものは不要と思う人もいそうなので、表示offの設定もあったほうがよいかと。

**Comment by kskwtnk** [comment]
@yoshi389111 

語彙がなさ過ぎて申し訳ないのですが、とても良さそうで是非やってみたいアイデアです！ありがとうございます！

> ちょっと頑張れば取れそうな感じ、かつもう少し頑張れば一つ上が狙える感じ

この雰囲気が醸成できたら非常に良さそうだと思っていて、参考にさせていただきます！

> 現状は人が少なくて、でも増えてほしいような分野のトロフィーがあると、結果的に全体の質もあがるのではないでしょうか。

こちらも仰る通り、活発に＆質の高いコミュニケーションが生まれると運営としてはとても嬉しいです！

現状だとこの役割はバッジ機能で提供しようと考えていますが、まだ実装してすぐなので機能的に不十分かつ認知度も足りていません
他にも実施したいな……と思っていたので、今回こういったご意見をいただけてとてもありがたいです！
